api/connectionmanager/client.go

```diff
@@ -196,3 +197,170 @@ func (store *ConnectionManager) TerminateConnectionsByUser(userID string) error
 
 	return err
 }
+
+// UEBA
+
+// UebaConfigurations get ueba configurations
+func (store *ConnectionManager) UebaConfigurations() (UebaConfigurations, error) {
+	configurations := UebaConfigurations{}
+	_, err := store.api.
+		URL("/connection-manager/api/v1/ueba/configure").
+		Get(&configurations)
+
+	return configurations, err
+}
+
+// SetUebaConfigurations set ueba configurations
+func (store *ConnectionManager) SetUebaConfigurations(configurations *UebaConfigurations) error {
+	_, err := store.api.
+		URL("/connection-manager/api/v1/ueba/configure").
+		Post(&configurations)
+
+	return err
+}
+
+// UebaAnomalySettings get ueba anomaly settings
+func (store *ConnectionManager) UebaAnomalySettings() (UebaAnomalySettings, error) {
+	settings := UebaAnomalySettings{}
+	_, err := store.api.
+		URL("/connection-manager/api/v1/ueba/anomaly-settings").
+		Get(&settings)
+
+	return settings, err
+}
+
+// CreateAnomalySettings create a host to host store
+func (store *ConnectionManager) CreateAnomalySettings(settings UebaAnomalySettings) error {
+
+	_, err := store.api.
+		URL("/connection-manager/api/v1/ueba/anomaly-settings").
+		Post(&settings)
+
+	return err
+}
+
+// StartAnalyzing start ueba analysis
+func (store *ConnectionManager) StartAnalyzing(datasetID string) error {
+	_, err := store.api.
+		URL("/connection-manager/api/v1/ueba/start-analyzing/%s", url.PathEscape(datasetID)).
+		Post(nil)
+
+	return err
+}
+
+// StopAnalyzing stop ueba analysis
+func (store *ConnectionManager) StopAnalyzing() error {
+	_, err := store.api.
+		URL("/connection-manager/api/v1/ueba/stop-analyzing").
+		Post(nil)
+
+	return err
+}
+
+// CreateIdForUebaScript create session ID for Ueba setup script
+func (store *ConnectionManager) CreateIdForUebaScript() (map[string]string, error) {
+	sessionId := make(map[string]string)
```

Comment:
You meant the struct to be model to be sth like this, right?:
```
type sessionID struct {
ID     string     `json:"id"
}
```
sure, I will change that.

I was not aware of the convention we follow here, so in the beginning, I weighed in between "using the map" or "using a struct model with a single field". In fact, I first created the struct model, but I ultimately got lured into using the map. 
My kind of reasoning was, 
1. The return type has only one "key-value" pair, so I thought maybe a map could just fit the case.
2. Since, this "struct" model will only be used once in the entire code. I didn't see the advantage we can get relating to reusability, so I kind of inclined towards the map.

But yes, I will change it.